dist/dist-packages/linux/openziti-controller/ziti-controller.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=OpenZiti Controller
+After=network-online.target
+
+[Service]
+# "ziti controller run" is the main process managed by this service and replaces entrypoint.bash
+Type=simple
+
+# manage the user and permissions for the service automatically
+DynamicUser=yes
+
+# allow binding low ports, e.g., 443/tcp
+AmbientCapabilities=CAP_NET_BIND_SERVICE
```

Comment:
should this be optional? I'd think most users will forward from their router or load balancer, and don't need to actually listen on `:443`

Reply:
For convenience, I'm leaning toward a set of defaults that provides the necessary privileges for common running states such as binding to privileged ports or running a tproxy router with `NET_ADMIN` capability (pertains to the router branch).

I could go either way, however. A more conservative default would be to leave this ambient grant as a comment.

Reply:
I think so, yes. I'd prioritized convenience, but on second thought, it's better to prioritize the default security posture and comment this grant out as a hint for the operator who does need to bind a privileged port.

That reminds me, too, that including a comment here about pass-through TLS for load balancing would be helpful.